components/spinner/spinner.js

```diff
@@ -1,22 +1,19 @@
 // Imports
+import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { getClassNamesWithMods } from '../_helpers';
 
 /**
  * General Spinner component. Use when you need spinner
  */
 function Spinner(props) {
-  const { size } = props;
-  const mods = props.mods ? props.mods.slice() : [];
+  const { className, size } = props;
```

Comment:
Check with Ivana if we should remove props

Reply:
Yeah, I guess we have decided that mods are kinda deprecated but still in place. They should be removed later.